packages/react-core/package.json

```diff
@@ -38,7 +38,7 @@
     "@patternfly/react-styles": "^4.11.8",
     "@patternfly/react-tokens": "^4.12.9",
     "focus-trap": "6.2.2",
-    "react-dropzone": "9.0.0",
+    "react-dropzone": "11.3.4",
```

Comment:
[`react-dropzone@11.0.0`](https://github.com/react-dropzone/react-dropzone/releases/tag/v11.0.0) is a breaking change for [AMQ](https://github.com/EnMasseProject/enmasse/blob/master/console/console-init/ui/src/components/UploadFile/UploadFile.tsx#L40) (will produce a type error for `File` vs `FileRejection`).

Openshift and Kogito also use `dropzoneProps` but don't rely on `onDropRejected` event types.

Reply:
Does it mean that I should leave the old version there?

Reply:
If you don't want to break AMQ leave it at 9.0.0. If you hear from @drcoolsanjeev that he's okay updating react-dropzone and changing the type [in AMQ](https://github.com/EnMasseProject/enmasse/blob/master/console/console-init/ui/src/components/UploadFile/UploadFile.tsx#L40) then you can upgrade to v11.